changelog.txt

```diff
@@ -1,2 +1,3 @@
-1.16.5-1.1.3:
+1.16.5-1.1.4:
+- Fix crash-inducing typo in SWEMInit. Fixes AlaharranHonor/SWLightingM#12
```

Comment:
If the repo is the same (issue) then you Can just do #12

Reply:
duh. sorry, my bad haha.